archives/levelObjects.md

```diff
@@ -23,8 +23,11 @@ Instance data of [Level Objects](../levelObjects/index.md) is stored as part of
     0010  [3]byte  Rotation Angles
     0013  byte     Unknown
     0014  byte     Object type
-    0015  int16    Unknown
-    0017  [4]byte  Unknown
+    0015  int16    Hitpoints
+    0017  byte  Unknown
+    0018  byte  State
+    0019  byte  Unknown
+    0020  byte  Unknown
```

Comment:
I'm not so happy with declaring the State already at this point. I know this was the classification in TSSHP and these values change in savegames. As long as we can't pinpoint the exact size (is it the one byte, or is it a 32bit value) as well as its values, I'm reluctant to declare it as State without further explanation.
Please revert the 4 bytes to Unknown or provide a description.